src/utils/helpers/browser-type-check/browser-type-check.spec.js

```diff
@@ -0,0 +1,20 @@
+import browserCheck from '.';
+
+describe('browserTypeCheck', () => {
+  it('returns true if "chrome" is true', () => {
+    const _window = { chrome: true, sidebar: false };
+    expect(browserCheck(_window)).toEqual(true);
+  });
+  it('returns true if "sidebar" is true', () => {
+    const _window = { chrome: false, sidebar: true };
+    expect(browserCheck(_window)).toEqual(true);
+  });
+  it('returns true if "chrome" and "sidebar" is true', () => {
+    const _window = { chrome: true, sidebar: true };
+    expect(browserCheck(_window)).toEqual(true);
+  });
+  it('returns true if neither "chrome" or "sidebar" is true', () => {
```

Comment:
`returns true` should be `returns false`.

Reply:
Yep good spot 👍

Reply:
:white_check_mark: